packages/util/src/indexeddb.ts

```diff
@@ -0,0 +1,132 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class IndexedDbDatabaseService {
```

Comment:
It might be worth considering using the [idb](https://www.npmjs.com/package/idb) library which is tiny and provides the convenient APIs instead of rolling our own solution.

Reply:
Done.